lib/metrics/prometheus.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
```

Comment:
I believe since 'prometheusMetric' and 'prometheusClient' can only be used together (same as the others), there's no need to put the 'metric' and 'client' in separate files, this can be a private thing within a 'prometheusClient'